test/wasm/signextend.js

```diff
@@ -6,7 +6,7 @@
 /* global assert,testRunner */ // eslint rule
 WScript.Flag("-WasmSignExtends");
 WScript.Flag("-WasmI64");
-WScript.LoadScriptFile("../UnitTestFrameWork/UnitTestFrameWork.js");
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");

```

Comment:
why this change?

Reply:
The change for the casing is necessary, however, we should still use forward slashes